tests/env.sh

```diff
@@ -135,6 +135,23 @@ check_versity_vars() {
 }
 
 check_user_vars() {
+  if [[ -z "$USERNAME_ONE" ]]; then
+    log 2 "missing USERNAME_ONE parameter"
+    return 1
+  fi
+  if [[ -z "$PASSWORD_ONE" ]]; then
+    log 2 "missing PASSWORD_ONE parameter"
+    return 1
+  fi
+  if [[ -z "$USERNAME_TWO" ]]; then
+    log 2 "missing USERNAME_ONE parameter"
+    return 1
+  fi
+  if [[ -z "$PASSWORD_TWO" ]]; then
+    log 2 "missing PASSWORD_ONE parameter"
+    return 1
+  fi
+  export USERNAME_ONE PASSWORD_ONE USERNAME_TWO PASSWORD_TWO
```

Comment:
Since these already exist (as you are checking if they are set) I don't think that you need to re-export.